### ISSUE TITLE
Error update GLPI 9.4.4 and Plugin 1.10.2 to 9.5.5 and 1.12.6

### ISSUE THREAD

**Issue by phamorais** [pluginsGLPI/fields#450]
Hey guys!

I'm trying to update the plugin in CLI and WEB, but I get the error below.

![image](https://user-images.githubusercontent.com/35368369/127227390-8240e88c-e443-421b-ba46-8dbc952855fe.png)

<center><table class='tab_cadre_fixe'><tr><th>Instalação de tabelas no MySQL<th></tr><tr class='tab_bg_1'><td align='center'>DBmysql::query() in /usr/share/nginx/html/pasc/inc/dbmysql.class.php line 309
  *** MySQL query error:
  SQL: CREATE TABLE IF NOT EXISTS `glpi_plugin_fields_fournineseventwozeroseventhreetwofielddropdowns` (
                  `id`                                      INT(11)        NOT NULL auto_increment,
                  `name`                                    VARCHAR(255)   DEFAULT NULL,
                  `completename`                            TEXT           DEFAULT NULL,
                  `comment`                                 TEXT           DEFAULT NULL,
                  `plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id` INT(11)        DEFAULT NULL,
                  `level`                                   INT(11)        DEFAULT NULL,
                  `ancestors_cache`                         TEXT           DEFAULT NULL,
                  `sons_cache`                              TEXT           DEFAULT NULL,
                  `entities_id`                             INT(11)        NOT NULL DEFAULT '0',
                  `is_recursive`                            TINYINT(1)     NOT NULL DEFAULT '0',
                  PRIMARY KEY                               (`id`),
                  KEY                                       `entities_id`  (`entities_id`),
                  KEY                                       `is_recursive` (`is_recursive`),
                  KEY                                       `plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id`
                                                            (`plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id`)
               ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
  Error: Incorrect table name 'glpi_plugin_fields_fournineseventwozeroseventhreetwofielddropdowns'
  Backtrace :
  ...ntwozeroseventhreetwofielddropdown.class.php:41
  plugins/fields/inc/dropdown.class.php:140          PluginFieldsFournineseventwozeroseventhreetwofieldDropdown::install()
  plugins/fields/inc/dropdown.class.php:45           PluginFieldsDropdown::create()
  plugins/fields/hook.php:80                         PluginFieldsDropdown::install()
  inc/plugin.class.php:726                           plugin_fields_install()
  inc/console/plugin/installcommand.class.php:134    Plugin->install()
  vendor/symfony/console/Command/Command.php:255     Glpi\Console\Plugin\InstallCommand->execute()
  vendor/symfony/console/Application.php:1000        Symfony\Component\Console\Command\Command->run()
  inc/console/application.class.php:249              Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:271         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:147         Symfony\Component\Console\Application->doRun()
  bin/console:102                                    Symfony\Component\Console\Application->run()
 SQL Error "1103": Incorrect table name 'glpi_plugin_fields_fournineseventwozeroseventhreetwofielddropdowns' in query "CREATE TABLE IF NOT EXISTS `glpi_plugin_fields_fournineseventwozeroseventhreetwofielddropdowns` (
                  `id`                                      INT(11)        NOT NULL auto_increment,
                  `name`                                    VARCHAR(255)   DEFAULT NULL,
                  `completename`                            TEXT           DEFAULT NULL,
                  `comment`                                 TEXT           DEFAULT NULL,
                  `plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id` INT(11)        DEFAULT NULL,
                  `level`                                   INT(11)        DEFAULT NULL,
                  `ancestors_cache`                         TEXT           DEFAULT NULL,
                  `sons_cache`                              TEXT           DEFAULT NULL,
                  `entities_id`                             INT(11)        NOT NULL DEFAULT '0',
                  `is_recursive`                            TINYINT(1)     NOT NULL DEFAULT '0',
                  PRIMARY KEY                               (`id`),
                  KEY                                       `entities_id`  (`entities_id`),
                  KEY                                       `is_recursive` (`is_recursive`),
                  KEY                                       `plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id`
                                                            (`plugin_fields_fournineseventwozeroseventhreetwofielddropdowns_id`)
               ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;"
Incorrect table name 'glpi_plugin_fields_fournineseventwozeroseventhreetwofielddropdowns'

![image](https://user-images.githubusercontent.com/35368369/127227620-199fd746-ee75-4100-a824-133d7d1a1178.png)

![image](https://user-images.githubusercontent.com/35368369/127227715-f57663fb-34cb-470b-ac60-df5433cae958.png)

Can help us?

**Comment by phamorais** [comment]
Any?

**Comment by phamorais** [comment]
@stonebuzz can u help us?

**Comment by phamorais** [comment]
Guys posting just to help in case anyone catches the same mistake.

In a joint analysis with the toline, we identified that in our base we have a fieds without a linked container, as shown in the image below

![image](https://user-images.githubusercontent.com/35368369/128209746-a7ebd190-d52c-4aff-a46d-da2b0ddb7c3a.png)

I cannot explain how this happened. But my solution was to delete the registry, and after that I was able to update without problems.

![image](https://user-images.githubusercontent.com/35368369/128209934-8fad499c-43c9-4dbd-8c50-a475718f56ae.png)

![image](https://user-images.githubusercontent.com/35368369/128209980-41599831-394d-44f8-a415-6fdd80ad3734.png)
